source/cache-proxy/setup-squid-as-a-transparent-proxy.rst

```diff
@@ -45,16 +47,18 @@ configured.
       is operational and tested.
    #. Click Save
 
-#. Click on the **Cache Management** tab.
+#. Click on the **Local Cache** tab.
```

Comment:
This **Local Cache** tab section needs to move up above the General tab settings. From a completely fresh install without squid, if you install the package it won't let you save on General until the Local Cache settings are done.